src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +20,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
Using CompletableFuture.allOf() is a good way to wait for all futures to complete, but you should also consider what to do if the computation throws an exception. You might want to handle it or at least log it to understand what went wrong.